src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    private StorageDao storage = new StorageDaoImpl();
+    private User validUser = new User();
+    private User validUser2 = new User();
+
+    @BeforeEach
+    public void each() {
+        storage.clear();
+
+        validUser.setId(null);
+        validUser.setLogin("123456");
+        validUser.setAge(18);
+        validUser.setPassword("123456");
+
+        validUser2.setId(null);
+        validUser2.setLogin("12345678");
```

Comment:
I believe you can use constants here as well

Reply:
Good point, thanks